<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SLRegression</class>
 <widget class="QDialog" name="SLRegression">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Linear Regression</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>404</x>
         <y>130</y>
         <width>201</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>Independent Variable(s)</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QListView" name="independenListView"/>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="DatasetLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>9</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose Dataset :</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>331</width>
         <height>231</height>
        </rect>
       </property>
       <property name="title">
        <string>Variables</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>352</x>
         <y>180</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>No dataset selected...</string>
        </property>
       </item>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>404</x>
         <y>60</y>
         <width>201</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Dependent Variable</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QListView" name="dependenListView"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>352</x>
         <y>85</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Statistics</string>
      </attribute>
      <widget class="QCheckBox" name="anovaCB">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Analysis Variance</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="cofficientsCB">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Coefficients</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="modelCB">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Model Summary</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="confidentICB">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Confident Interval</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="covMatCB">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Covariance Matrix</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>271</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Residual</string>
       </property>
       <widget class="QCheckBox" name="resDurbinCB">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Durbin-Watson</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SLRegression</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
